spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/discovery/reactive/ConsulReactiveDiscoveryClient.java

```diff
@@ -75,11 +76,14 @@ public String description() {
 	}
 
 	private List<HealthService> getHealthServices(String serviceId) {
-		HealthServicesRequest request = HealthServicesRequest.newBuilder().setTag(this.properties.getDefaultQueryTag())
```

Comment:
I just realized that the Reactive client was still suffering from #680 . This brings it inline w/ the new behaviour.